tests/integration/source_code/test_jobs.py

```diff
@@ -122,10 +122,8 @@ def test_job_task_linter_library_installed_cluster(
 
 def test_job_linter_some_notebook_graph_with_problems(simple_ctx, ws, make_job, make_notebook, make_random, caplog):
     expected_messages = {
-        'second_notebook:3 [dbfs-usage] Deprecated file system path: /mnt/something',
-        'second_notebook:3 [implicit-dbfs-usage] The use of default dbfs: references is deprecated: /mnt/something',
-        'some_file.py:0 [dbfs-usage] Deprecated file system path: /mnt/foo/bar',
-        'some_file.py:0 [implicit-dbfs-usage] The use of default dbfs: references is deprecated: /mnt/foo/bar',
+        'some_file.py:0 [direct-filesystem-access] The use of direct filesystem references is deprecated: /mnt/foo/bar',
```

Comment:
please rebase, see https://github.com/databrickslabs/ucx/commit/f7d793d126e0b3f8af37a41a948013ea81340b9d#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5, and update the documentation in root's README.md by removing DBFS usage references and adding doc section for this newly added linter